Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -555,16 +555,20 @@ def query_logs_command(args: dict, client: Client) -> Tuple[str, Dict[str, List[
     return human_readable, ec, raw_results
 
 
-def get_table_name(records: List[Dict[str, Any]]):
+def get_table_name(query: str) -> str:
     """
     Table name is stored in log_type attribute of the records
     Args:
-        records: Records to get the table name of
+        query: Query string, i.e SELECT * FROM firewall.threat LIMIT 1
 
     Returns:
-        The records table name
+        The query's table name
     """
-    return next(record.get('log_type', {}).get('value') for record in records if record.get('log_type')) or ''
+    FIND_TABLE_NAME_FROM_QUERY = r'(FROM `)(\w+.\w+)(`)'
```

Comment:
global consts should be at the top of the code

Reply:
But I only use it once in this method's scope 
Should I lowercase it?
or just use it as value in re.search?
Seems to much to make a const of something that's being used only once

Reply:
yea so lets lower case it

Reply:
Done